README.md

```diff
@@ -40,12 +40,20 @@ An example for a "porcelain" API would be [ROS2](https://www.ros.org/). Others a
 | [RTA-VRTE](https://www.etas.com/en/products/rta-vrte.php) | [Adaptive AUTOSAR](https://www.autosar.org/standards/adaptive-platform/) platform software framework for vehicle computer from [ETAS GmbH](https://www.etas.com) |
 | [Cyclone DDS](https://github.com/eclipse-cyclonedds/cyclonedds) | Performant and robust open-source DDS implementation maintained by [ADLINK Technology Inc.](https://www.adlinktech.com/) |
 
-### Supported Platforms
+### Targeted quality levels & platforms
 
- * Linux
- * QNX
- * macOS (initial support - please test and report bugs)
- * Windows 10 (not yet - currently in progress)
+> [Quality level](./CONTRIBUTING.md#quality-levels) are 5..1, where 1 is highest
+
+|CMake project/target   | QNX  | Linux, Windows, MacOS | Comment                             |
+|-----------------------|:----:|:---------------------:|:-----------------------------------:|
+| example_icedelivery   | 4    | 4                     |                                     |
+| example_iceperf       | 4    | 4                     |                                     |
+| example_singleprocess | 4    | 4                     |                                     |
+| iceoryx_dds           | 4    | 4                     |                                     |
+| iceoryx_meta          | 4    | 4                     |                                     |
+| iceoryx_posh          | 1, 2 | 4                     | Will be split into separate targets |
```

Comment:
For Linux, I would assume this to be at least a 3, don't you think so?
And since we want to support all tier 1 targets or ROS, the same should apply for Windows and MacOS. Maybe by adding compiler and QAC warnings to level 2.

Reply:
Good point, but I'd suggest to extend level 4 instead of switch to level 3. Should we document the compilers in the CONTRIBUTING.md where we say which targets we have?

Reply:
you teased me with 4 levels on line 45 but didn't specify it. Maybe 4 could become 5 and some parts of 3 could be moved to 4

Reply:
Done